make-it/EmMakefile

```diff
@@ -4,31 +4,43 @@
 # and instead uses the version on PATH (ie that is unix mkdir compatible)
 MKDIR="mkdir"
 
-VIREO_SDK_DIR=..
+VIREO_SDK=..
 
-OBJS=$(VIREO_SDK_DIR)/make-it/objs
-INCDIR=$(VIREO_SDK_DIR)/source/include
-BIN=$(VIREO_SDK_DIR)/bin
-TARGETDIR=$(VIREO_SDK_DIR)/dist
-CORESOURCEDIR=$(VIREO_SDK_DIR)/source/core
-IOSOURCEDIR=$(VIREO_SDK_DIR)/source/io
-UTSOURCEDIR=$(VIREO_SDK_DIR)/source/unittest
+TARGETS_SUPPORTED=asmjs-unknown-emscripten wasm32-unknown-emscripten
```

Comment:
We should add support for native builds also. Now, native builds create "Debug" and "Release" directories under "dist".

Reply:
yep, as discussed in the summary: This pull request covers only the emscripten part of the proposal

Native will be updated by whoever wants to tackle it in the future. The proposal is here: https://github.com/ni/VireoSDK/issues/328